quickstarts/gradle/thorntail/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-bin.zip
```

Comment:
Was this tried with a more recent version of Gradle?

Reply:
Yes, I tried with 7.2 version but it seemed like this plugin is not compatible

```
Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details
> Task :thorntail-package FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Some problems were found with the configuration of task ':thorntail-package' (type 'PackageTask').
  - In plugin 'org.wildfly.swarm.plugin.gradle.ThorntailArquillianPlugin' type 'org.wildfly.swarm.plugin.gradle.PackageTask' property 'mainClassName' is private and annotated with @Input, @Optional.
    
    Reason: Annotations on private getters are ignored.
    
    Possible solutions:
      1. Make the getter public.
      2. Annotate the public version of the getter.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#private_getter_must_not_be_annotated for more details about this problem.
  - In plugin 'org.wildfly.swarm.plugin.gradle.ThorntailArquillianPlugin' type 'org.wildfly.swarm.plugin.gradle.PackageTask' property 'packaging' is private and annotated with @Input.
    
    Reason: Annotations on private getters are ignored.
    
    Possible solutions:
      1. Make the getter public.
      2. Annotate the public version of the getter.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#private_getter_must_not_be_annotated for more details about this problem.
  - In plugin 'org.wildfly.swarm.plugin.gradle.ThorntailArquillianPlugin' type 'org.wildfly.swarm.plugin.gradle.PackageTask' property 'projectArtifactFile' is private and annotated with @InputFile.
    
    Reason: Annotations on private getters are ignored.
    
    Possible solutions:
      1. Make the getter public.
      2. Annotate the public version of the getter.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#private_getter_must_not_be_annotated for more details about this problem.
  - In plugin 'org.wildfly.swarm.plugin.gradle.ThorntailArquillianPlugin' type 'org.wildfly.swarm.plugin.gradle.PackageTask' property 'propertiesFile' is private and annotated with @Optional, @InputFile.
    
    Reason: Annotations on private getters are ignored.
    
    Possible solutions:
      1. Make the getter public.
      2. Annotate the public version of the getter.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#private_getter_must_not_be_annotated for more details about this problem.
  - In plugin 'org.wildfly.swarm.plugin.gradle.ThorntailArquillianPlugin' type 'org.wildfly.swarm.plugin.gradle.PackageTask' property 'propertiesFromFile' is private and annotated with @Input.
    
    Reason: Annotations on private getters are ignored.
    
    Possible solutions:
      1. Make the getter public.
      2. Annotate the public version of the getter.
    
    Please refer to https://docs.gradle.org/7.2/userguide/validation_problems.html#private_getter_must_not_be_annotated for more details about this problem.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 8s
4 actionable tasks: 4 executed

```